README.md

```diff
@@ -1,2 +1,4 @@
-# microvers_floating_position
-this is a microverse floating and positioning of element
+# project : FLOATING AND POSITIONING [Newyork Times](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)
+This is the third project floating and positioning in html and Css curriculum at [Microverse](https://www.microverse.org/) 
+- The assignment is [here](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements)
```

Comment:
**[REQUIRED]**
Nice job adding a description to the readme file. But it's also a best practice to add the name and git account of both contributor's